app/src/main/res/layout/fragment_patient_chart.xml

```diff
@@ -86,4 +86,11 @@
       android:layout_height="fill_parent"
       android:layout_below="@+id/patient_chart_last_update_section"/>
 
-</RelativeLayout>
+  <ListView
+    android:id="@+id/chart_list"
+    android:layout_width="200dp"
+    android:layout_height="match_parent"
+    android:layout_gravity="left|start"
```

Comment:
This should just be `start`. `left` and `start` mean the same thing for left-to-right languages, but they're a contradiction for right-to-left (RTL) languages. If we ever localise for RTL languages, we want the drawer to show up on the right-hand side :)